### PR TITLE
Simplify favicon SVG.

### PR DESCRIPTION
**Pull request by Thom1729**
This should result in no visible change.

- Remove the IDs, which are unused.
- Remove the `version` attribute, which is deprecated.
- Remove the `height` and `width` attributes and the `<title>` element, which are useless for a favicon.
- Apply the red fill directly to the `A` path and remove the class/style.
- Remove the `translate` transformation and fix the path to work without it.

### CODE REVIEW

**Comment by Thom1729** [comment]
Published at https://spec.yaml.io/favicon/.

File: www/build/favicon.svg [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
   viewBox="0 0 512 470.647">
-  <title id="title-001">YAML logo</title>
-  <defs>
-    <style>.cls-1{fill:#cb171e;}</style>
-  </defs>
-  <polygon id="L" points="512 422.735 395.638 422.735 395.638 250.125 347.442 250.125 347.442 469.647 512 469.647 512 422.737 512 422.735" />
-  <polygon id="M" points="87.701 250.177 87.701 470.647 135.004 470.647 135.004 318.569 184.509 420.789 221.743 420.789 272.939 314.976 272.939 470.602 318.318 470.602 318.318 250.177 256.358 250.177 201.381 349.883 149.021 250.177 87.701 250.177 87.701 250.177" />
-  <path id="A" class="cls-1" d="M330.294,195.39H228.433l-20.717,50.024H162.61L257.99,20.465h46.137l91.51,224.949h-48.2L330.293,195.39Zm-16.92-44.911-31.226-82.55-34.837,82.55h66.063Z" transform="translate(0 -19.939)" />
-  <polygon id="Y" points="235.793 0 143.978 137.674 143.978 224.949 87.702 224.949 87.702 137.674 0 0 63.25 0 119.018 88.646 175.243 0 235.793 0 235.793 0" />
+  <polygon points="512 422.735 395.638 422.735 395.638 250.125 347.442 250.125 347.442 469.647 512 469.647 512 422.737 512 422.735" />
```
Comment (ingydotnet) [inline www/build/favicon.svg:5]: Can we add the `id='.'` attributes back and put the lines in Y A M L order?

**Review by ingydotnet — CHANGES_REQUESTED** [review]

File: www/build/favicon.svg (hunk 1 above)
Comment (Thom1729) [reply www/build/favicon.svg:5]: We *could*. Why?

Comment (ingydotnet) [reply www/build/favicon.svg:5]: afaict, that's the only human indication of what each lines means.

Comment (Thom1729) [reply www/build/favicon.svg]: Done.

**Review by ingydotnet — APPROVED** [review]

**Pull request merged**